migtests/tests/analyze-schema/dummy-export-dir/schema/tables/table.sql

```diff
@@ -13,7 +13,7 @@ CREATE TABLE salaries2 (
 	from_date timestamp NOT NULL,
 	to_date timestamp NOT NULL,
 	PRIMARY KEY (emp_no,from_date)
-) PARTITION BY RANGE (((from_date)::date - '0001-01-01bc')::integer) ;
+) PARTITION BY RANGE (extract(epoch from date(from_date))) ; --this expression is correct now, earlier it wasn't appropriate as PG syntax 
```

Comment:
Ques: this comment is also coming up in the schema dump?

Reply:
yeah as it in the Stmt, will remove it this is just for reviewing purpose only, to mention the reason there as to why I am changing the DDL